Scripts/Interfaces/IInstantiateMenuUI.cs

```diff
@@ -9,13 +9,24 @@ namespace UnityEditor.Experimental.EditorVR
 	/// </summary>
 	public interface IInstantiateMenuUI
 	{
+	}
+
+	public static class IInstantiateMenuUIMethods
+	{
+		internal static Func<Transform, IMenu, GameObject> instantiateMenuUI { get; set; }
+
 		/// <summary>
 		/// Instantiate custom menu UI on a proxy
-		/// Transform = Ray origin
-		/// IMenu = Prefab (with IMenu component) to instantiate
-		/// Returns an instantiated UI GameObject
 		/// </summary>
-		Func<Transform, IMenu, GameObject> instantiateMenuUI { set; }
+		/// <param name="rayOrigin">The ray origin of the proxy that this menu is being instantiated from</param>
+		/// <param name="menuPrefab">The prefab (with an IMenu component) to instantiate</param>
```

Comment:
Not sure what you mean on this one. The param names match the parameters and are in the correct order.